src/core/src/resources/poet_config.toml

```diff
@@ -69,3 +69,8 @@ genesis_batch_path = ''
 # engine.
 
 validator_pub_key = '/etc/sawtooth/keys/validator.pub'
+
+# log_dir specifies the directory where the log file must be located
+# Note that the log filename would be poet-consensus.log( not configurable )
```

Comment:
[Question]: Why not make it configurable? Can we accept complete path of file as input in config file?

Reply:
Does the C++ logs also logged in this path ? Did we check